packages/client.js

```diff
@@ -7,5 +7,6 @@
  */
 
 import { Client } from '../src/client/client.js';
+import { Debug } from '../src/client/debug/debug.js';
 
-export { Client };
+export { Client, Debug };
```

Comment:
client.js does not have any React dependencies. This should probably go in packages/react.js.

Reply:
updated - moved to react.js